partiql-parser/src/lexer.rs

```diff
@@ -741,6 +755,8 @@ impl<'input> fmt::Display for Token<'input> {
             Token::ExpReal(txt) => write!(f, "<{}:REAL>", txt),
             Token::Real(txt) => write!(f, "<{}:REAL>", txt),
             Token::String(txt) => write!(f, "<{}:STRING>", txt),
+            Token::Time(txt) => write!(f, "<{}>", txt),
```

Comment:
nit: should we add some additional annotation to the displayed value? Perhaps `:TIME`:

```suggestion
            Token::Time(txt) => write!(f, "<{}:TIME>", txt),
```

Reply:
In this case the type-name get included in the `txt` by the lex function, e.g. `TIME(30)` will be `<TIME:30>` (see L895-L896 in the diff).